experiments/graphql-tools-mocking.md

```diff
@@ -21,3 +21,35 @@ You should be able to see the result of the query that was executed directly.
 This demo also includes a running server at http://0.0.0.0:4000/graphql which
 is running GraphiQL. This allows you to explore the original schema while
 querying with the same fake data.
+
+## Benchmark
+
+Install hyperfine with brew or similar:
+
+```sh
+$ brew install hyperfine
+```
+
+Copy the schemas that you want to test against to `schemas/` folder.
+
+Run the tests in your environment and export it to MD file like:
+
+```sh
+$ hyperfine --warmup 3 -r 10 \
+  'ts-node -T graphql-tools-mocking/benchmark.ts graphql-tools-mocking.graphql' \
+  'ts-node -T graphql-tools-mocking/benchmark.ts <api-1>.graphql' \
+  'ts-node -T graphql-tools-mocking/benchmark.ts <api-2>.graphql' \
+  --export-markdown benchmark-result-<architecture>-<with-server/without-server>.md
```

Comment:
What exactly are we measuring?  Is it time to boot + mock 1 query?

Reply:
yes, in this case it is warm boot tho. The point here was not to measure the performance of mocking 1 query, but test the performance impact of big schemas and, as they grow, will this cause problems for our development productivity.